README.md

```diff
@@ -1,2 +1,51 @@
 # wg-meta
-An apporach to add metadata to the main wireguard config
+An approach to add metadata to the main wireguard config, written in Perl.
+
+## Configuration
+There is only one config file: `wg-meta.yaml` which defines the additional (meta-) attributes.:
+```yaml
+# according to wg-meta.schema.json
+default_attributes:
+  name:
+    mandatory: true
+  alias:
+    mandatory: true
+custom_attributes:
+- name: some_custom_name
+  mandatory: true
+- name: some_other_custom_attribute
+  mandatory: false
```

Comment:
one option would be to have a prefix for custom attributtes ...
in openapi for example you can have custom attributes with an x- prefix

a schema file for the contract does not have to be changed when the contract gets changed ...

I can provide that if you want